main.js

```diff
@@ -77,20 +82,23 @@ function startActivity() {
   showTimer();
 }
 
-// new function
+
 function showTimer() {
   removeClass(startActivityForm);
   fixTime(currentActivity.minutes, currentActivity.seconds)
   activityHeader.innerText = 'Current Activity';
   userDescriptionInput.innerText = currentActivity.description;
-  if (currentActivity.category === 'Study') {
-    startButton.classList.add('study-active')
-  } else if (currentActivity.category === 'Meditate') {
-    startButton.classList.add('meditate-active')
-  } else if (currentActivity.category === 'Exercise') {
-    startButton.classList.add('exercise-active')
+  changeActivityColor(startButton, 'Study')
+  changeActivityColor(startButton, 'Meditate')
+  changeActivityColor(startButton, 'Exercise')
+}
```

Comment:
we could replace 91-31 with:
addClass(startButton, `${currentActivity.category.toLowerCase()}`)